awx/main/models/unified_jobs.py

```diff
@@ -728,6 +728,13 @@ class Meta:
         editable=False,
         help_text=_("The Collections names and versions installed in the execution environment."),
     )
+    ansible_version = models.CharField(
+        max_length=50,
```

Comment:
Probably better to err on this side of making this larger, in case some future version of Ansible prints a really lengthy version string e.g.,

```python
>>> len('ansible automation engine [core enterprise edition 4.0.1beta]')
61
```

Reply:
https://www.youtube.com/watch?v=7ffj8SHrbk0